packages/wonder-blocks-birthday-picker/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "@khanacademy/wonder-blocks-birthday-picker",
+    "version": "0.0.1",
+    "design": "v1",
+    "publishConfig": {
+      "access": "public"
+    },
+    "description": "",
+    "main": "dist/index.js",
+    "module": "dist/es/index.js",
+    "source": "src/index.js",
+    "scripts": {
+      "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "dependencies": {
+      "@babel/runtime": "^7.16.3",
+      "@khanacademy/wonder-blocks-color": "^1.1.20",
+      "@khanacademy/wonder-blocks-core": "^4.2.0",
+      "@khanacademy/wonder-blocks-dropdown": "^2.6.1",
+      "@khanacademy/wonder-blocks-icon": "^1.2.24",
+      "@khanacademy/wonder-blocks-layout": "^1.4.6",
+      "@khanacademy/wonder-blocks-spacing": "^3.0.5",
+      "@khanacademy/wonder-blocks-typography": "^1.1.28"
```

Comment:
I think it can be any either peer or regular deps. That's how we have handle it with the other existing WB components (at least for the internal wb packages).